### ISSUE TITLE
Influx: Label filtering support in explore logs visualization

### ISSUE THREAD

**Issue by marefr** [grafana/grafana#17501]
**What would you like to be added**:
The primary way to filter logs result should be the labels mechanism (currently via the labels column, but in the future possibly via a row expansion) by clicking on the label (tag) itself. This would use the existing label functionality in logs visualization that Loki uses.

**Why is this needed**:
It's preferred if the adhoc tag filters are the secondary way of filtering.

**Comment by jcdauchy** [comment]
Is it possible to use influxdb as a log datasource in Explore / Logpanel ? If this is the case, any input how to set this up ?

**Comment by aocenas** [comment]
@jcdauchy Yes, there were some bugs that were fixed in 6.6 so it may be an issue of you hitting those bugs. Please try now, you should not need any special setup. If it does not work please file a separate issue.

**Comment by aocenas** [comment]
@marefr As we have the log row detail UI can we close this? I think that addresses this issue.

**Comment by marefr** [comment]
@aocenas I guess, but adhoc filtering for elasticsearch/infludb logs are yet not supported which was part of this. Either close this and open a new one or keep this open and change the title maybe?

**Comment by aocenas** [comment]
@marefr Ah you are right it work in loki only at the moment. Will change the title.
